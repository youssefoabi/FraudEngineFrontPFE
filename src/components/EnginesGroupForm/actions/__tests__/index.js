import { itShouldTestActionCreators } from '@cdiscount/jest-utils';

import * as actions from '../';

describe('EnginesGroup Form actions', () => {
  itShouldTestActionCreators(actions);
});
