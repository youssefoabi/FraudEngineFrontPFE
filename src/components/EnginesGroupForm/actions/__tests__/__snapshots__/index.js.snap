// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnginesGroup Form actions CREATE_ENGINESGROUP_ERROR 1`] = `"CREATE_ENGINESGROUP_ERROR"`;

exports[`EnginesGroup Form actions CREATE_ENGINESGROUP_REQUEST 1`] = `"CREATE_ENGINESGROUP_REQUEST"`;

exports[`EnginesGroup Form actions CREATE_ENGINESGROUP_SUCCESS 1`] = `"CREATE_ENGINESGROUP_SUCCESS"`;

exports[`EnginesGroup Form actions EDIT_ENGINESGROUP_ERROR 1`] = `"EDIT_ENGINESGROUP_ERROR"`;

exports[`EnginesGroup Form actions EDIT_ENGINESGROUP_REQUEST 1`] = `"EDIT_ENGINESGROUP_REQUEST"`;

exports[`EnginesGroup Form actions EDIT_ENGINESGROUP_SUCCESS 1`] = `"EDIT_ENGINESGROUP_SUCCESS"`;

exports[`EnginesGroup Form actions RESET_ENGINESGROUP_SAVE_ERROR 1`] = `"RESET_ENGINESGROUP_SAVE_ERROR"`;

exports[`EnginesGroup Form actions RESET_ENGINESGROUP_SAVE_SUCCESS 1`] = `"RESET_ENGINESGROUP_SAVE_SUCCESS"`;
